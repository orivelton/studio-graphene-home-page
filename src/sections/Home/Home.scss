@import '../../assets/scss/helper.scss';

.home {
  display: flex;
  height: 100vh;
  background: url('../../assets/img/bck-home.svg') no-repeat center 0 $primary;
  justify-content: center;
  align-items: center;
}

.big-title {
  color: $white;
  display: flex;
  flex-direction: column;
  align-items: center;
  &__top {
    font-size: 82px;
    &--large {
      display: flex;
      font-size: 115px;
    }
  }
}

.request-info {
  position: fixed;
  background: $white;
  text-decoration: none;
  transform: rotate(-90deg);
  font-size: 18px;
  color: $night-rider;
  padding: 9.4px 9.4px 3.51px 8.49px;
  right: 0;
  top: 327px;
  z-index: 1;
}
