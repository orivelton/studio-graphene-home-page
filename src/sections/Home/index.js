import React from 'react';
import Logo from '../../components/Logo/Logo';
import Stamp from '../../components/Stamp';

const Home = () => (
  <>
    <Logo />
    <Stamp textTop={'Gormmet Experience'} text={'Logo'} textBottom={'Gormmet Experience'} />
  </>

);

export default Home;
