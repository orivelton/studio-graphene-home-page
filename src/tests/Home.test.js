import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../section/Home';

test('renders correctly Home', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
