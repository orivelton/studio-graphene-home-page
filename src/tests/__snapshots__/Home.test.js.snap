// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Home 1`] = `
<section
  className="section home"
>
  <div
    className="count-section"
  >
    <span
      className="current-section current-section--double"
    >
      01
    </span>
    <span
      className="current-section"
    />
    <span
      className="current-section"
    />
  </div>
  <div
    className="big-title uppercase"
  >
    <div
      className="stamp uppercase icon iconstamp"
    >
      <p
        className="stamp__text"
      >
        Logo
      </p>
    </div>
    <h1
      className="big-title__top"
    >
      The Best Foodie
      <span
        className="big-title__top--large"
      >
        Experience
      </span>
    </h1>
    <h2>
      Now in London
    </h2>
    <a
      className="request-info uppercase"
      href="/"
      title="Request more Info"
    >
      Request Info
    </a>
  </div>
</section>
`;
